### ISSUE TITLE
Highlighting doesn't work in VSCodium

### ISSUE THREAD

**Issue by DeflateAwning** [Gruntfuggly/todo-tree#834]
The `TODO`, `FIXME`, etc. keywords don't get highlighted white in VS Codium the same way they do in VS Code. Not certain what the difference is caused by, but it seems to be consistent.

**Comment by GitMensch** [comment]
Are you sure that you use the same version of the extension? I guess the issue is that you may use an outdated version, see #423.

**Comment by DeflateAwning** [comment]
Good catch; this is the issue. The version installed with VS Codium is v0.0.215.
